### PR TITLE
chore: add Wrangler worker env for Josh

### PR DESCRIPTION
**Pull request by joshJarr**
- Simply updated wrangler.toml with my a development environment var for Josh.

### CODE REVIEW

File: packages/api/wrangler.toml
Comment (flea89) [inline packages/api/wrangler.toml:137]: Would you mind adding the `GATEWAY_URL` as well?

**Review by flea89 — COMMENTED** [review]
Nit. 
update the PR title to chore: ~A~add Wrangler worker env for Josh.. 

(the PR name is used as a default commit message once you squash & merge the PR in main, and I think we generally don't capitalise).


This looks good, apart from the 2 small comments.

File: packages/api/wrangler.toml
Comment (joshJarr) [reply packages/api/wrangler.toml]: 👍  Done, should I also add `GATEWAY_URL = "https://ipfs.io" ` to the template at the bottom of this file? Or is it only required for some configs?

Comment (flea89) [reply packages/api/wrangler.toml]: Good catch, yes please 👍

Comment (joshJarr) [reply packages/api/wrangler.toml]: Cool, added now, thanks!

**Review by flea89 — APPROVED** [review]
LGTM!

**Comment by olizilla** [comment]
@joshJarr @flea89 if you are happy with this PR, then one of you should merge it!

in general we encourage the person opening the PR to merge their own work once they have some approval from another team member, and are confident that the change will land cleanly. This usually means also being ready to test the change on staging and then merging the release PR to get it into prod. Not so relevant with this change, I'm just verbalising the process.

**Comment by joshJarr** [comment]
Thanks for explaining the process @olizilla! I'll merge this in now.
Cheers all!

**Pull request merged**
